client/my-sites/plugins/plugins-discovery-page/upgrade-nudge.jsx

```diff
@@ -148,10 +148,11 @@ const UpgradeNudge = ( { siteSlug, paidPlugins } ) => {
 			callToAction={ translate( 'Upgrade to Business' ) }
 			icon="notice-outline"
 			showIcon={ true }
-			href={ pluginsPlansPageFlag ? pluginsPlansPage : `/checkout/${ siteSlug }/business` }
```

Comment:
No action needed, just adding a note for the future:

`pluginsPlansPageFlag` seems to be always false since it picks its value from a config flag that does not exist.

https://github.com/Automattic/wp-calypso/blob/dff510cf252239e8949e88aedabe1e19218ab0e7/client/my-sites/plugins/plugins-discovery-page/upgrade-nudge.jsx#L53